src/components/StageBar.vue

```diff
@@ -38,31 +38,51 @@ export default {
 @import '@/styles/theme.scss';
 
 .stage-bar {
-  display: flex;
-  flex-flow: row wrap;
-  justify-content: center;
-  padding-left: 16px;
-
+  position: relative;
+  display: grid;
+  grid-template-columns: auto auto auto;
```

Comment:
IMHO this component should be more generic and be able to render N amount of tabs. Doing this, change the  the stages will be smoother. Currently, if we add a new tab it will be placed in a new row.